src/main/java/com/jagex/runescape/frame/console/Console.java

```diff
@@ -12,6 +12,7 @@
 import com.jagex.runescape.media.renderable.actor.Player;
 import com.jagex.runescape.scene.SceneCluster;
 import com.jagex.runescape.scene.tile.WallDecoration;
+import tech.henning.fourthreefive.Configuration;
```

Comment:
Shouldn't we rename the package to something like org.runejs.client.Configuration?